<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EdgesDialog</class>
 <widget class="QDialog" name="EdgesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hide common edges</string>
  </property>
  <property name="windowIcon">
   <iconset theme="bookmark-new"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>688</width>
        <height>445</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="fieldsLayout"/>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="allButton">
       <property name="text">
        <string>Show all</string>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EdgesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>581</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EdgesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>708</x>
     <y>596</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>allButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>EdgesDialog</receiver>
   <slot>allButtonChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>597</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>allButtonChecked(bool)</slot>
  <slot>searchBoxEdited(QString)</slot>
 </slots>
</ui>
